opensync-ext-static/src/main/resources/ProfileRf.json

```diff
@@ -0,0 +1,86 @@
+{
+	"model_type": "Profile",
+	"id": 9,
+	"customerId": 2,
+	"profileType": "rf",
+	"name": "DefaultRf-2g",
+	"details": {
+		"model_type": "RfConfiguration",
+		"rf": "Default-RF-2g",
+		"profileType": "rf",
+		"rfConfigMap": {
```

Comment:
Maybe I misunderstood in the meeting but I thought the discussion was to have a 2.4G profile and a single 5G profile to cover all types of 5G

Reply:
My understanding was that the RF profile will have a separate config in the rfConfigMap for each radio that is valid, which are currently 2.4G and 5G in a 2 radio AP, or 2.4G, 5G-L, 5G-U in a 3 radio AP. Then any call to get a certain parameter should be for a given radioType as well.

Reply:
Ralph is correct, you need to be able to define on a per radioType basis. On APs which have 2 5GHz radios, you may well want to have different values